_data/team.yml

```diff
@@ -208,3 +208,22 @@
   type: mc
   social:
     - {name: "twitter", link: "https://twitter.com/garricn"}
+
+-
+  id: "17"
```

Comment:
@ajpinedam please make sure these ID numbers are unique inside this file. There is already an `id: 17`.

Reply:
@ZevEisenberg 9 and 10 are also repeated you want me to fix these too?

Reply:
Oh, yes, please!